plugins/active_directory_ldap/komand_active_directory_ldap/util/api.py

```diff
@@ -165,6 +165,28 @@ def delete(self, conn, dn: str):
     def manage_user(self, conn, dn: str, status: bool):
         return ADUtils.change_account_status(conn, dn, status, self.logger)
 
+    @with_connection
+    def manage_users(self, conn, dns: List[str], status: bool) -> dict:
```

Comment:
Can we type hint for the `conn` variable here?

Reply:
It's not explicitly used